crates/header-translator/src/method.rs

```diff
@@ -618,7 +618,12 @@ impl fmt::Display for Method {
 
         // Arguments
         for (param, arg_ty) in &self.arguments {
-            write!(f, "{}: {arg_ty},", handle_reserved(param))?;
+            let param = if param == "_" {
+                String::from("_")
+            } else {
```

Comment:
Huh, do you have an example of when this happens?

Reply:
I think the only place it happens is in the argument for [`NSSetUncaughtExceptionHandler`](https://docs.rs/icrate/latest/icrate/Foundation/fn.NSSetUncaughtExceptionHandler.html). The reason I have to handle it specially is because `heck` converts the `"_"` to an empty string `""`.

Reply:
Ah okay, let's keep it as-is for now then